src/pool.c

```diff
@@ -3463,7 +3463,11 @@ miner_on_submit(json_object *message, client_t *client)
     if (BN_cmp(hd, bd) >= 0)
     {
         /* Yay! Mined a block so submit to network */
-        log_info("+++ MINED A BLOCK +++");
+        log_info("+++ MINED A BLOCK +++ %.5s %"PRIu64" / %"PRIu64 " %"PRIu64,
+                 client->address,
+                 pool_stats.round_hashes,
+                 pool_stats.network_hashrate,
```

Comment:
You're comparing different things here: round difficulty and network hashrate. You should use `network_difficulty`, not `network_hashrate`.